shed/event_streams.py

```diff
@@ -254,33 +257,31 @@ def curate_streams(self, nds, outbound):
         If we get multiple streams make (name, (doc, doc, doc, ...))
         Otherwise (name, (doc,))
         """
+        # if there are multiple streams
+        if isinstance(nds[0], tuple):
```

Comment:
I'm not certain how you got that input, it is not `(name, doc)` nor `(name, (doc, doc...))`, so it may not be applicable?

If it is:
I think that `(('start', {}), )` would (should?) be normalized to `('start', {})` on the way out.
Similarly on the way in it would become `('start', ({}, ))`. I can add a test for this tomorrow.